h/templates/admin/organizations_edit.html.jinja2

```diff
@@ -0,0 +1,14 @@
+{% extends "h:templates/layouts/admin.html.jinja2" %}
+
+{% set page_id = 'organizations_edit' %}
+{% set page_title = 'Edit organization' %}
+
+{% block content %}
+  {{ form }}
+  {% if delete_url %}
```

Comment:
The positioning of this delete button is a bit wonky for me:

![image](https://user-images.githubusercontent.com/439947/38216577-352856f6-3699-11e8-8509-05e9c8da055f.png)

Reply:
Yep. Zero points for beauty here. My plan was to get it in production and solicit some suggestions, hoping that we can fix the other views (OAuth clients, groups) at the same time.